parquet-avro/src/main/java/org/apache/parquet/avro/AvroRecordConverter.java

```diff
@@ -963,7 +1003,8 @@ static boolean isElementType(Type repeatedType, Schema elementSchema) {
       // synthetic wrapper. Must be a group with one optional or required field
       return true;
     } else if (elementSchema != null && elementSchema.getType() == Schema.Type.RECORD) {
-      Schema schemaFromRepeated = CONVERTER.convert(repeatedType.asGroupType());
+      AvroSchemaConverter converter = new AvroSchemaConverter(true, conf);
```

Comment:
As the description of `CONVERTER` states, it is only for testing whether the generated schema contains 2-level lists. I think, it would be safe to simply hard-code for that `AvroSchemaConverter` constructor to enable reading `INT96` values to not to throw an exception. As a result, you don't have to push down the conf all the way down.